settings.gradle

```diff
@@ -75,21 +75,21 @@ dependencyResolutionManagement {
             def antlrVersion = version "antlr", "4.13.0"
             // WARNING: When upgrading to a version of bytebuddy that supports a new bytecode version,
             // make sure to remove the now unnecessary net.bytebuddy.experimental=true in relevant CI jobs (Jenkinsfile).
-            def byteBuddyVersion = version "byteBuddy", "1.14.18"
-            def classmateVersion = version "classmate", "1.5.1"
+            def byteBuddyVersion = version "byteBuddy", "1.15.10"
```

Comment:
Hey @gavinking could you please file a Jira issue next time you upgrade a dependency -- especially Bytebuddy? It's a very sensitive dependency that impacts our support of versions of OpenJDK, so worth mentioning in changelogs in particular.